test/integrations/destinations/zoho/router/account.ts

```diff
@@ -0,0 +1,128 @@
+import { upsertPayload1V2, upsertPayload2V2, commonConnectionConfigV2 } from '../../zoho/common';
+import { defaultMockFns } from '../../zoho/mocks';
+
+const commonDestinaitonConfig = {
```

Comment:
We need to consider the following scenarios:
1. Region is available in both account and destination.config in presence of accountDefinition
2. Account has options but doesn't have accountDefinition
3. Account has definition but no options. Fallback should consider destination.config but add log.